start-demo.sh

```diff
@@ -0,0 +1,19 @@
+export OZONE_DIR=$PWD/ozone && \
+mkdir -p $OZONE_DIR
+# Download the project && \
+export VERSION=1.0.0-SNAPSHOT && \
+./mvnw org.apache.maven.plugins:maven-dependency-plugin:3.2.0:get -DremoteRepositories=https://nexus.mekomsolutions.net/repository/maven-public -Dartifact=net.mekomsolutions:ozone-distro:$VERSION:zip -Dtransitive=false --legacy-local-repository && \
+./mvnw org.apache.maven.plugins:maven-dependency-plugin:3.2.0:unpack -Dproject.basedir=$OZONE_DIR -Dartifact=net.mekomsolutions:ozone-distro:$VERSION:zip -DoutputDirectory=$OZONE_DIR/ozone-distro-$VERSION
+
+export DISTRO_PATH=$OZONE_DIR/ozone-distro-$VERSION;  \
```

Comment:
Could this be made prettier?
There is no need for `;` and `\` for instance, since we are not copying it in a shell.